src/back/TicketManager/src/Core/TicketManager.Core.Services/Migrations/20230417182304_AddSeatReservation.cs

```diff
@@ -0,0 +1,186 @@
+                oldRowVersion: true);
+
+            migrationBuilder.AlterColumn<byte[]>(
+                name: "RowVersion",
+                table: "Events",
+                type: "bytea",
+                rowVersion: true,
+                nullable: false,
+                defaultValue: new byte[0],
+                oldClrType: typeof(byte[]),
+                oldType: "bytea",
+                oldRowVersion: true);
+
+            migrationBuilder.AlterColumn<byte[]>(
+                name: "RowVersion",
+                table: "Accounts",
+                type: "bytea",
+                rowVersion: true,
+                nullable: false,
+                defaultValue: new byte[0],
+                oldClrType: typeof(byte[]),
+                oldType: "bytea",
+                oldRowVersion: true);
+
+            migrationBuilder.CreateTable(
+                name: "Sectors",
+                columns: table => new
+                {
+                    Id = table.Column<Guid>(type: "uuid", nullable: false),
+                    EventId = table.Column<Guid>(type: "uuid", nullable: false),
+                    Name = table.Column<string>(type: "character varying(250)", maxLength: 250, nullable: false),
+                    PriceInSmallestUnit = table.Column<int>(type: "integer", nullable: false),
+                    NumberOfColumns = table.Column<int>(type: "integer", nullable: false),
+                    NumberOfRows = table.Column<int>(type: "integer", nullable: false),
+                    DateModified = table.Column<DateTime>(type: "timestamp with time zone", nullable: false),
+                    RowVersion = table.Column<byte[]>(type: "bytea", rowVersion: true, nullable: false, defaultValue: new byte[0])
+                },
+                constraints: table =>
+                {
+                    table.PrimaryKey("PK_Sectors", x => x.Id);
+                });
+
+            migrationBuilder.CreateTable(
+                name: "SeatReservation",
+                columns: table => new
+                {
+                    Id = table.Column<Guid>(type: "uuid", nullable: false),
+                    SectorId = table.Column<Guid>(type: "uuid", nullable: false),
+                    UserId = table.Column<Guid>(type: "uuid", nullable: false),
+                    ReservedSeatNumber = table.Column<int>(type: "integer", nullable: false),
+                    CreationDate = table.Column<DateTime>(type: "timestamp with time zone", nullable: false)
+                },
+                constraints: table =>
+                {
+                    table.PrimaryKey("PK_SeatReservation", x => x.Id);
+                    table.ForeignKey(
+                        name: "FK_SeatReservation_Sectors_SectorId",
+                        column: x => x.SectorId,
+                        principalTable: "Sectors",
+                        principalColumn: "Id",
+                        onDelete: ReferentialAction.Cascade);
+                });
```

Comment:
Here you should write sql that copy old sectors to new sectors table. If you don't want to do that, fine we don't have production database. If we had production database it would be unacceptable. At least write sql that delete all events to make our database consistent.